docs/community-toolkit/ravendb.md

```diff
@@ -0,0 +1,299 @@
+
+builder.AddProject<Projects.ExampleProject>()
+       .WithReference(ravenServer)
+       .WaitFor(ravenServer);
+```
+
+The data volume persists RavenDB data outside the lifecycle of its container. The data volume is mounted at the `/var/lib/ravendb/data` path in the RavenDB container and when a `name` parameter isn't provided, the name is generated at random. For more information on data volumes and details on why they're preferred over [bind mounts](#add-meilisearch-resource-with-data-bind-mount), see [Docker docs: Volumes](https://docs.docker.com/engine/storage/volumes).
+
+### Add RavenDB server resource with data bind mount
+
+To add a data bind mount to the RavenDB server resource, call the `Aspire.Hosting.RavenDBBuilderExtensions.WithDataBindMount` method:
+
+```csharp
+var builder = DistributedApplication.CreateBuilder(args);
+
+var ravenServer = builder.AddRavenDB("ravenServer")
+                         .WithDataBindMount(source: @"C:\RavenDb\Data");
+
+builder.AddProject<Projects.ExampleProject>()
+       .WithReference(ravenServer)
+       .WaitFor(ravenServer);
+```
+
+[!INCLUDE [data-bind-mount-vs-volumes](../includes/data-bind-mount-vs-volumes.md)]
+
+Data bind mounts rely on the host machine's filesystem to persist the RavenDB data across container restarts. The data bind mount is mounted at the `C:\RavenDb\Data` on Windows (or `/RavenDB/Data` on Unix) path on the host machine in the RavenDB container. For more information on data bind mounts, see [Docker docs: Bind mounts](https://docs.docker.com/engine/storage/bind-mounts).
+
+### Add secured RavenDB server resource  
+
+To create a new secured RavenDB instance using settings from a pre-configured _settings.json_ file or a self-signed certificate, use the `RavenDBServerSettings.Secured` method or `RavenDBServerSettings.SecuredWithLetsEncrypt` for Let’s Encrypt configurations. These methods allow you to specify the domain URL, certificate details, and additional server settings.
+Here’s an example of how to add a secured RavenDB server resource using Let's Encrypt:
+
+```csharp
+var builder = DistributedApplication.CreateBuilder(args);
+
+var serverSettings = RavenDBServerSettings.SecuredWithLetsEncrypt(
+    domainUrl: "https://mycontainer.development.run",
+    certificatePath: "/etc/ravendb/security/cluster.server.certificate.mycontainer.pfx");
+
+var ravendb = builder.AddRavenDB("ravenSecuredServer", serverSettings)
+    .WithBindMount("C:/RavenDB/Server/Security", "/etc/ravendb/security", false)
+    .AddDatabase("ravendbSecured");
+
+builder.AddProject<Projects.ExampleProject>()
+       .WithReference(ravendb)
+       .WaitFor(ravendb);
+```
+> [!IMPORTANT]  
+> Ensure the certificate path is accessible to the container by bind-mounting it to `/etc/ravendb/security`.
+
+### Hosting integration health checks
+
+The RavenDB hosting integration automatically adds a health check for the RavenDB server resource. It verifies that the RavenDB server is running and reachable.
```

Comment:
=>
.... for the RavenDB server resource, verifying that the server is running and reachable.